.vscode/settings.json

```diff
@@ -53,6 +56,7 @@
     "quetzallabs",
     "rehype",
     "reqs",
+    "retryable",
```

Comment:
this is not a word (the correct spelling is retriable, but canRetry is probably better anyways; see below)

```suggestion
```

Reply:
<img width="755" alt="image" src="https://github.com/user-attachments/assets/f4b16eae-f9c2-497f-b193-a98d5f95f3f4" />

Sure I will change it to canRetry